src/Symfony/Component/Form/FormInterface.php

```diff
@@ -244,6 +244,13 @@ public function isSynchronized();
     public function getTransformationFailure();
 
     /**
+     * Returns the data transformation failure of the children, if any.
+     *
+     * @return array
+     */
+    public function getChildrenTransformationFailures();
```

Comment:
@xabbuh what about adding this method to the class Form only? Since button won't have it anyway....